ui/src/components/TakeDealDialog.svelte

```diff
@@ -17,7 +17,6 @@
   const WS_ADDRESS = 'ws://127.0.0.1:5001/ws'
```

Comment:
can you change this to `ws://127.0.0.1:${SWAPD_PORT}/ws` where `SWAPD_PORT` is an environment variable? there's also a function for this `readSwapdPortNumber()` in utils/rpcApi.ts you may be able to use

Reply:
yeah it is! I think this was the only place it was still hard-coded, so this should fix #127 also